modules/dynamic/src/smithy4s/dynamic/internals/DynamicModelCompiler.scala

```diff
@@ -216,7 +216,7 @@ private[dynamic] object Compiler {
       update(id, shape.traits, bytes)
 
     override def enumShape(id: ShapeId, shape: EnumShape): Unit = {
-      val values: List[EnumValue[Int]] =
+      val values: List[EnumValue[String]] =
```

Comment:
Another meaningful change : now the string enumerations are represented as a raw string in the dynamic data. This allows for fulfilling the new `EnumTag` contract